tests/integration/test_apply_checks.py

```diff
@@ -4309,6 +4311,7 @@ def test_apply_checks_all_row_checks_as_yaml_with_streaming(ws, make_schema, mak
                 datetime(2025, 1, 12, 3, 0, 0),
                 {"key1": 1},
                 {"field1": 1},
+                "255.255.255.255",
```

Comment:
This makes `test_apply_checks_all_row_checks_as_yaml_with_streaming` to fail. Update it so that it falls within the CIDR: '192.168.1.0/24', e.g. "192.168.1.3",

Also values from `col9` must also be added to the `expected`.